src/opm/parser/eclipse/Parser/ParserItem.cpp

```diff
@@ -287,26 +291,38 @@ void ParserItem::push_backDimension( const std::string& dim ) {
         return m_description;
     }
 
-    void ParserItem::setDescription(std::string description) {
+
+    void ParserItem::setSizeType(item_size size_type) {
+        this->m_sizeType = size_type;
+    }
+
+
+    void ParserItem::setDescription(const std::string& description) {
         m_description = description;
     }
 
 bool ParserItem::operator==( const ParserItem& rhs ) const {
-    if( !( this->type           == rhs.type
-        && this->m_name         == rhs.m_name
-        && this->m_description  == rhs.m_description
-        && this->m_sizeType     == rhs.m_sizeType
-        && this->m_defaultSet   == rhs.m_defaultSet ) )
-            return false;
+    if( !( this->data_type      == rhs.data_type
+           && this->m_name         == rhs.m_name
+           && this->m_description  == rhs.m_description
+           && this->input_type     == rhs.input_type
+           && this->m_sizeType     == rhs.m_sizeType
+              && this->m_defaultSet   == rhs.m_defaultSet ) )
```

Comment:
What happened to the indentation level here (`&& this->m_defaultSet`)?

Reply:
fixed